content-repo/extra-docs/articles/troubleshooting.md

```diff
@@ -78,6 +78,53 @@ Key | Value
 --- |  ----
 `python.pass.extra.keys` | `--network=host`
 
+After you add the server configuration, run the `/reset_containers` command from the Cortex XSOAR CLI to reset all containers and to begin using the new configuration.
+
 **Notes:**
```

Comment:
perhaps convert to callout?

Reply:
I think it is ok. It looks clear in the doc.